doc/fake-devicegraphs-yaml-format.md

```diff
@@ -136,12 +153,28 @@ Example:
   beginning of that extended partition, so the last one of those logical
   partitions might have size "unlimited" again.
 
+  size may be missing and defaults to 'unlimited'.
+
+  Note: all size values are exact. So, for example, on a 100 GiB disk you
+  can't create a 100 GiB partition but only one with 1023 MiB.
```

Comment:
I got lost. Didn't you mean "on a 1 GiB disk you can't create a 1 GiB partition but only one with 1023 MiB"? 100GiB would be 102400 MiB. Isn't it?

On the other hand, I'm not sure if it's obvious for all readers why the maximum size is 1023.